templates/collection.liquid

```diff
@@ -1,9 +1,93 @@
+<section id="collection">
+  <div class="container">
+    <div class="grid-row">
+
+{% paginate collection.products by 24 %}
+
 {% if collection.all_products_count > 0 %}
-{% for product in collection.products %}
-<h2><a href="{{ product.url }}">{{ product.title }}</a></h2>
-{{ product.description }}
-<img src="{{ product.featured_image | product_img_url: 'grande' }}" alt="{{ product.title | escape }}" />
-{% endfor %}
+
+  {% if collection.url != blank %}
+
+      <div class="grid-item item-s-12 item-m-auto grid-row no-gutter align-content-start">
+        <div class="grid-item item-s-6 item-m-12 margin-bottom-tiny">
+          {% comment %}Types List{% endcomment %}
+          <h3 class="font-bold font-uppercase font-size-small">Type</h3>
+          <ul>
+          {% for product_type in collection.all_types %}
+          <li>
+            <a href="{{ collection.url }}/{{ product_type | handle }}">{{ product_type }}</a>
+          </li>
+          {% endfor %}
+          </ul>
+        </div>
+        {% comment %}Brands List{% endcomment %}
+        <div class="grid-item item-s-6 item-m-12">
+          <h3 class="font-bold font-uppercase font-size-small">Brand</h3>
+          <ul>
+          {% for product_vendor in collection.all_vendors %}
+            <li>
+          		<a href="{{ collection.url }}/{{ product_vendor | handle }}">{{ product_vendor }}</a>
+            </li>
+          {% endfor %}
+          </ul>
+        </div>
+      </div>
+
+  {% endif %}
+
+      <div class="grid-item flex-grow no-gutter grid-row">
+
+        {% comment %}Breadcrumbs{% endcomment %}
+        <div class="grid-item item-s-12 font-bold font-size-small font-uppercase">
+          {% if current_tags != blank %}
+          <a href="{{ collection.url }}">{{ collection.handle }}</a> > {{ current_tags }}
```

Comment:
I feel that the `>` for breadcrumb doesnt suit the design, mainly because with this type it sits a bit too low. Will we ever got 3 deep on the breadcrumb? If not I would suggest `:` or even `—`

Reply:
its a placeholder

Reply:
i'll use a dash